app/src/main/java/com/example/unsplashphotos/domain/usecase/PhotoUseCase.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.unsplashphotos.domain.usecase
+
+
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import androidx.paging.PagingSource
+import com.example.unsplashphotos.data.model.Photo
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+
+class PhotoUseCase @Inject constructor(private val photoPagingSource: PagingSource<Int, Photo>) {
+
+    private val config = PagingConfig(pageSize = 20, enablePlaceholders = false)
+
+    fun fetchPhotos(): Flow<PagingData<Photo>> {
```

Comment:
Android Paging library shouldn't reside inside UseCase. It's place is in data layer. Please refactor accordingly after going through the UseCase articles.